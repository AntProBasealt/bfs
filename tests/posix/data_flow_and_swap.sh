bfs_diff basic \! -type f -a -type d
