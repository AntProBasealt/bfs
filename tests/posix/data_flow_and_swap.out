basic
basic/c
basic/e
basic/g
basic/g/h
basic/i
basic/j
basic/k
basic/k/foo
basic/l
basic/l/foo
basic/l/foo/bar
